ruby/README.md

```diff
@@ -34,3 +34,116 @@ There are the official Ruby bindings for Apache Arrow.
 [Red Parquet](https://github.com/apache/arrow/tree/master/ruby/red-parquet) is the Parquet bindings.
 
 
+# Cookbook
```

Comment:
Could you add one more heading-level for all cookbook related headings like:

```suggestion
## Cookbook
```

This file already has the top-level heading: "# Apache Arrow Ruby"